xml/System/Guid.xml

```diff
@@ -1260,10 +1313,55 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1;netstandard-1.0;netstandard-1.1;netstandard-1.2;netstandard-1.3;netstandard-1.4;netstandard-1.5;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">An object to compare to this instance.</param>
+        <summary>Compares this instance to a specified <see cref="T:System.Guid" /> object and returns an indication of their relative values.</summary>
+        <returns>A signed number indicating the relative values of this instance and <paramref name="value" />.
+          <list type="table">
+            <listheader>
+              <term>Return value</term>
+              <description>Description</description>
+            </listheader>
+            <item>
+              <term>A negative integer</term>
+              <description>This instance is less than <paramref name="value" />.</description>
+            </item>
+            <item>
+              <term>Zero</term>
+              <description>This instance is equal to <paramref name="value" />.</description>
+            </item>
+            <item>
+              <term>A positive integer</term>
+              <description>This instance is greater than <paramref name="value" />.</description>
+            </item>
+          </list>
+        </returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Guid.CompareTo%2A> method compares the GUIDs as if they were values provided to the <xref:System.Guid.%23ctor%28System.Int32%2CSystem.Int16%2CSystem.Int16%2CSystem.Byte%5B%5D%29> constructor, as follows:
```

Comment:
```suggestion
This member is an explicit interface member implementation. It can only be used when the <xref:System.Guid" /> instance is cast to an <xref:System.IComparable> interface. 

The `CompareTo` method compares the GUIDs as if they were values provided to the <xref:System.Guid.%23ctor%28System.Int32%2CSystem.Int16%2CSystem.Int16%2CSystem.Byte%5B%5D%29> constructor, as follows:
```